bindata/assets/alerts/client-go-requests.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: client-go
+  namespace: openshift-kube-apiserver
+spec:
+  groups:
+    - name: control-plane-client-go
+      rules:
+        - record: client_go_request_duration:method_get:histogram_quantile99
+          expr: |
+            histogram_quantile(0.99,
+              sum(rate(
+                label_replace(rest_client_request_duration_seconds_bucket{verb="GET"},
+                  "url_host","$1$2","url","https?://([^:^/]*)(:[0-9]+)?.*?")[5m:30s]
```

Comment:
I'd prefer `host` as a label instead of `url_host`, `url` feels unnecessary since the metric is about requests

Reply:
I think that I choose that because I was afraid `host` already existed as label

Reply:
updated to use host